### PR TITLE
QOLDEV-1015 prepare for CKAN 2.11

### PR DESCRIPTION
**Pull request by ThrawnCA**
CKAN 2.11 scenario tests don't yet pass, but add them in reporting-only mode and fix some of the incompatibilities, including replacing obsolete test containers.

Drop support for Python 2 including CKAN 2.8 (and Celery), because it's not easily compatible with newer containers (and is long obsolete anyway).

### CODE REVIEW

File: .ahoy.yml
Comment (duttonw) [inline .ahoy.yml:1]: I know ahoy looks nice but is another tool you need to install on a linux system, an ideas if we could just go to plain shell scripts?

File: bin/serve.sh [hunk 1]
```diff
@@ -1,23 +1,9 @@
 #!/usr/bin/env sh
 set -e
 
-dockerize -wait tcp://postgres:5432 -timeout 1m
```
Comment (duttonw) [inline bin/serve.sh:4]: since dockerize has been removed, should we still be installing it?

**Review by duttonw — APPROVED** [review]

File: bin/serve.sh (hunk 1 above)
Comment (ThrawnCA) [reply bin/serve.sh:4]: We still use it in `ahoy.yml`

File: .ahoy.yml
Comment (ThrawnCA) [reply .ahoy.yml:1]: It's totally possible. Thus far it hasn't been worth the bother to migrate away.

**Pull request merged**